gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureGeneratorTask.java

```diff
@@ -51,6 +52,13 @@ public void execute(Task _task) {
         });
     }
 
+    // Set the path sensitivity of the sources, which would otherwise default to ABSOLUTE
+    @Override
+    @PathSensitive(PathSensitivity.RELATIVE)
+    public FileTree getSource() {
```

Comment:
There's a failing ErrorProne check about this being extensible but not having a JavaDoc to explain what to do with it. I'm not sure whether to prefer leaving this extensible so people can override the path sensitivity in subclasses, or just making this final and keeping things simpler.

Reply:
Everything else is final, this method likely should be as well

Reply:
I made this final.